public/js/test/mochitest/head.js

```diff
@@ -58,6 +87,17 @@ function _afterDispatchDone(store, type) {
   });
 }
 
+/**
+ * Wait for a specific action type to be dispatch.
+ * If an async action, will wait for it to be done.
+ *
+ * @memberof mochitest/waits
+ * @param {Object} dbg
+ * @param {String} type
+ * @param {Number} eventRepeat
+ * @return {Promise}
+ * @static
+ */
```

Comment:
Hmm, I wonder if we can make this file a flow file and then git the params mostly for free. This would be nice because we could type `dbg` and we would then have typed tests. Don't feel like you have to do this for this work to be merged, but I don't think we need the params because we should eventually get them from flow